src/main/java/org/fogbowcloud/saps/engine/core/catalog/exceptions/CatalogException.java

```diff
@@ -0,0 +1,10 @@
+package org.fogbowcloud.saps.engine.core.catalog.exceptions;
+
+public class CatalogException extends RuntimeException{
+
+    private static final long serialVersionUID = -2520888793776997437L;
+
+    public CatalogException(String msg){
```

Comment:
same problem here. In fact, you've just made a `WeslleyProblem`

Reply:
Done